packages/firestore-compat/test/fields.test.ts

```diff
@@ -387,8 +387,7 @@ apiDescribe('Timestamp Fields in snapshots', (persistence: boolean) => {
 });
 
 apiDescribe('`undefined` properties', (persistence: boolean) => {
-  const settings = { ...DEFAULT_SETTINGS };
-  settings.ignoreUndefinedProperties = true;
+  const settings = { ...DEFAULT_SETTINGS, ignoreUndefinedProperties: true };
```

Comment:
This change to `fields.test.ts` is the fix from https://github.com/firebase/firebase-js-sdk/pull/7108 patched in.